lib/react_client/react_interop.dart

```diff
@@ -423,10 +425,26 @@ class ReactDartInteropStatics2 implements ReactDartInteropStatics {
 /// See [ReactDartInteropStatics], [createReactDartComponentClass].
 class ComponentStatics<T extends Component> {
```

Comment:
@aaronlademann-wf mentioned this above, but can we remove this generic parameter now that this class will never be used by `Component2`?

Reply:
Indeed!